### PR TITLE
Omits peripheral metrics (internal, host OS and few k8s system metrics by default)

### PR DESCRIPTION
**Pull request by mmichael**

### CODE REVIEW

File: wavefront/values.yaml [hunk 1]
```diff
@@ -80,8 +80,28 @@ collector:
   #   env: production
   #   region: us-west-2
 
+  ## If set to true, host OS metrics will be collected
+  hostOSMetrics: false
+
   ## Filters to apply towards all metrics collected by the collector
   filters:
+    # Optimized metrics collection to focus collection on default dashboard support.
```
Comment (vikramraman) [inline wavefront/values.yaml:88]: Maybe reword this to:

`# Optimized metrics collection to omit peripheral metrics.`

**Comment by vikramraman** [comment]
lgtm. Just a minor tweak to the comment.

File: wavefront/values.yaml (hunk 1 above)
Comment (mmichael) [reply wavefront/values.yaml]: Done

**Review by vikramraman — APPROVED** [review]
